base/gocb_dcp_feed.go

```diff
@@ -45,6 +45,22 @@ func getHighSeqMetadata(bucket Bucket) ([]DCPMetadata, error) {
 	return metadata, nil
 }
 
+func getCollectionIDs(bucket Bucket, scope, collection *string) ([]uint32, error) {
+	c, ok := bucket.(*Collection)
+	if !ok {
```

Comment:
It looks like getCollectionIDs (and by extension, StartGocbDCPFeed) only work if bucket is actually a *Collection.  Is that correct?  If so, should we change the bucket parameter on StartGocbDCPFeed to be *Collection, and have callers responsible for casting as needed?

Reply:
Migrated a bunch of code to Collection.